python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -448,7 +471,7 @@ def run(self, *, exposures, id_generator=None):
         result : `lsst.pipe.base.Struct`
             Results as a struct with attributes:
```

Comment:
Is the results Struct guaranteed to have all of the documented attributes, now?

Reply:
I'm not sure what you mean?

If the task completes, it returns `result`. If an exception is raised, the input parameter has (hopefully!) been modified, but nothing is returned.

Reply:
Oh, but apparently I never added docs for the `result` parameter: I'll do that now.

Reply:
Here's what I added:

```
        result : `lsst.pipe.base.Struct`, optional
            Result struct that is modified to allow saving of partial outputs
            for some failure conditions. If the task completes successfully,
            this is also returned.
```